<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortDialog</class>
 <widget class="QDialog" name="SerialPortDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>465</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>465</width>
    <height>370</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>350</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>350</height>
      </size>
     </property>
     <property name="title">
      <string>串口参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="DataByte">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="RTS_checkBox">
        <property name="text">
         <string>RTS</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>校验位：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>停止位：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>数据位：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="StopBit">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>串口名：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QPushButton" name="Openserial_Button">
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="PortName"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="BaudRate">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="ParityCheck">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>显示</string>
     </property>
     <widget class="QLabel" name="RplayDisplaylabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>231</width>
        <height>111</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="Input_textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>231</width>
        <height>111</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="SendData_pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="BinaryData_radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>二进制</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="HexData_radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>十六进制</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>233</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>继电器命令：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Command_comboBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>空</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>查询八路状态</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>查询指令返回信息</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第一路开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制返回信息</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第一路关</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制返回信息</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第二路开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第二路关</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第三路开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第三路关</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第四路开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第四路关</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第五路开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第五路关</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第六路开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第六路关</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第七路开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第七路关</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第八路开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>控制第八路关</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>查询八路光耦状态</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>查询返回信息</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>全开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>全断</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>闪开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>闪断</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
